docs/package.json

```diff
@@ -20,8 +20,10 @@
     "marked": "^2.0.0",
     "next": "^12.0.1",
     "react": "^17.0.1",
+    "react-cookie": "^4.1.1",
     "react-dom": "^17.0.1",
-    "react-live": "^2.2.2"
+    "react-live": "^2.2.2",
+    "universal-cookie": "^4.0.4"
```

Comment:
@manvoid do we need `universal-cookie` to be mentioned explicitly? If not, can we remove this line?

- `universal-cookie` is a dependency of `react-cookie` so it would automatically get installed: https://www.npmjs.com/package/react-cookie.
- If we update `react-cookie` we would also need to update `universal-cookie`, which is not clear if we separate them out.